app/uk/gov/hmrc/gform/views/govuk_wrapper.scala.html

```diff
@@ -176,7 +176,8 @@
     @hmrcReportTechnicalIssue(
       ReportTechnicalIssue(
         serviceCode = appConfig.contactFormServiceIdentifier,
-        language = Some(language.code)
+        language = Some(language.code),
+        classList = Some("govuk-body")
```

Comment:
I'm not sure this is correct. Issue is about delete link in file-upload component.

Reply:
The font for Delete link is coming form govuk-body class and that applies to all the elements in the body. However, the report technical issues link is outside the body, while it should have been inside. Putting it inside means copying it into every template and deleting it from the govuk_template. This is a quick fix to make all links consistent

Reply:
@josef-vlach I have added the govuk-link to the Delete anchor. Please take a look

Reply:
Had a chat with @richardhattonUX on the changes and he seems happy with it

Reply:
@girishkamat Look's good, but can we remove `delete-file` class, it seems have no effect.

Reply:
@josef-vlach It looks like some javascript code uses it to lookup the elements

`    function init () {
      $('.govuk-file-upload').on('change', handleFileUpload);
      $('.uploaded-files').on('click', '.delete-file', handleFileDelete);
    }`